workspaces/ui-v2/src/entry-points/local/local-cli.tsx

```diff
@@ -54,9 +56,27 @@ const appConfig: OpticAppConfig = {
     documentation: {
       allowDescriptionEditing: true,
     },
+    api: {
+      domain:
+        (window.location.hostname.indexOf('useoptic.com') >= 0
+          ? process.env.REACT_APP_PROD_API_BASE
+          : process.env.REACT_APP_STAGING_API_BASE) || 'https://api.o3c.info',
+    },
+    sharing: {
+      enabled: true,
+      specViewerDomain:
+        (window.location.hostname.indexOf('useoptic.com') >= 0
```

Comment:
Wouldn't this always return false? Since the local cli will be run against `localhost` which will always run in staging?

In what cases would we want to upload to prod and what cases would we want to upload to staging?

Reply:
Oh wait derp thank you, you're totally right. This should probably actually select whether the frontend is running in dev mode or in a release build

Reply:
Do you think it's better to just have a single env variable that gets injected per build? or two separate ones that look at release status